mastermind.rb

```diff
@@ -0,0 +1,139 @@
+# Command Line Mastermind Game
+class Mastermind
+  attr_reader :player_score, :master_score, :player_guess,
+              :game_count, :board, :guess_count, :guess
+
+  private
+
+  attr_writer :master_code, :player_score, :master_score,
+              :player_guess, :game_count, :board, :guess_count,
+              :guess
+
+  def initialize
+    set_master_code
+    reset_board
+    reset_scores
+    reset_guess_count
+    display_rules
+    start_game
+  end
+
+  def reset_guess_count
+    @guess_count = 0
+  end
+
+  def set_master_code
+    @master_code = Array.new(4) { rand(2...8) }
```

Comment:
Do we have something against 1's and 9's lol!

Reply:
Haha! the 0's and 1's are responses from the mastermind and the codes should be 6 "colors" so I represented them with 6 numbers

Reply:
👍